src/api/app/controllers/webui/projects/rebuild_times_controller.rb

```diff
@@ -60,7 +59,7 @@ def call_diststats(bdep, jobs)
         f.write(bdep)
         f.close
         f = File.open(indir + '/_jobhistory.xml', 'w')
-        f.write(jobs.dump_xml)
```

Comment:
since you are here, would you mind to use the block format for ```open``` and ```write``` instead of having to call f.close manually?

Reply:
like this?

```
-        f = File.open(indir + '/_builddepinfo.xml', 'w')
-        f.write(bdep)
-        f.close
-        f = File.open(indir + '/_jobhistory.xml', 'w')
-        f.write(jobs)
-        f.close
+        File.open(indir + '/_builddepinfo.xml', 'w') { |f| f.write(bdep) }
+        File.open(indir + '/_jobhistory.xml', 'w') { |f| f.write(jobs) }

Reply:
I'm just afraid @Ana06 will want me to name it file_to_write_to

Reply:
maybe File.join instead of + '/...'

Reply:
at least `file`... `f` can be many things...